root/work/edit_form.tt

```diff
@@ -99,6 +99,18 @@
 
   <div class="documentation">
     [%- iswc_bubble(link_entity(work)) -%]
+
+    <div class="bubble" id="type-bubble">
+      <span id="type-bubble-default">
+        [%- l('Select any type from the list to see its description. If the work
+               doesn\'t seem to match any type, just leave this blank.') -%]
```

Comment:
Should be `doesn’t` as [typographically-correct punctuation is preferred](https://musicbrainz.org/doc/Style/Miscellaneous).